### PR TITLE
Made microsoft session address non-readonly

### PR DESCRIPTION
**Pull request by rileymckenna**
Made microsoft session address non-readonly

### CODE REVIEW

File: src/SDKs/_metadata/peering_resource-manager.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+2020-02-27 20:33:18 UTC
```
Comment (isra-fel) [inline src/SDKs/_metadata/peering_resource-manager.txt]: This file should be in `/eng/mgmt/mgmtmetadata/` and seems to be in old format. Please follow the guide to [update your build tools](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#updating-build-tools) then generate again.
Thanks

File: sdk/peering/Microsoft.Azure.Management.Peering/src/Microsoft.Azure.Management.Peering.csproj [hunk 2]
```diff
@@ -7,13 +7,11 @@
...
+		<Version>2.0.1-preview</Version>
 		<PackageTags></PackageTags>
-    <PackageReleaseNotes><![CDATA[
-		- Added Registered ASN operations.
-		- Added Registered Prefix operations.
-		- Added Improvement to Peering Service operations. 
-		]]>
+    <PackageReleaseNotes>
+		- Made microsoft session address non-readonly
+
```
Comment (isra-fel) [inline sdk/peering/Microsoft.Azure.Management.Peering/src/Microsoft.Azure.Management.Peering.csproj]: Please keep the CDATA tag
```suggestion
    <PackageReleaseNotes><![CDATA[
- Made microsoft session address non-readonly
]]>
```

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @rileymckenna  please see my inline comments, and the file `sdk/peering/Microsoft.Azure.Management.Peering/src/Generated/code-model-v1` I suppose it's checked in by accident?

**Comment by rileymckenna** [comment]
@isra-fel ive fixed the build issues and updated the meta data properly

**Review by isra-fel — APPROVED** [review]
There's a known limitation of latest `autorest` that it would produce an extra (redundant) file `code-model-v1`. If you see it you can just remove it next time. Thanks

**Comment by rileymckenna** [comment]
@isra-fel Ready to merge

**Pull request merged**